docs/jsonapi-spec.textile

```diff
@@ -589,6 +644,12 @@ h4(#commandUpdateOneOptions). updateOne Command Options
 |_. Option |_. Type |_. Description |
 | @upsert@ | Boolean | When @true@ if no documents match the @filter@ clause the command will create a new _empty_ document and apply the @update@ clause to the empty document. If the @_id@ field is included in the @filter@ the new document will use this @_id@, otherwise a random value will be used see "Upsert Considerations":#considerationsUpsert for details. When false the command will only update a document if one matches the filter. Defaults to @false@.|
 
+h4(#commandUpdateOneFailure). updateOne Multi Document Failure Modes
```

Comment:
how is update one considering multi document failure when it targets one document? same for insertOne, findOne, etc